File: 01-introduccion/ejemplos-if-else.ts
//EJEMPLOS IF & ELSE

// 1

const calificacion = 5;
if(calificacion >= 0 && calificacion < 4.5){
    console.log("reprobado")
}else{
    if(calificacion >= 4.5 && calificacion < 7){
        console.log("supletorio")
    }else{
        if(calificacion >= 7 && calificacion <= 10){
            console.log("aprobado")
        }else{
            console.log("ingrese numeros entre 0 y 10")
        }
    }
}

// 2

const estaLloviendo = true;
const noHiceLaTarea = true;

if(estaLloviendo == true && noHiceLaTarea == true){
    console.log("no voy a clases")
}else{
    if(estaLloviendo != true && noHiceLaTarea == true){
        console.log("invento una excusa")
    }else{
        if(estaLloviendo == true && noHiceLaTarea != true){
            console.log("llevo paraguas")
        }else{
            console.log("voy a clases")
        }
    }
}

// 3

const miroUnaPelicula = true;
const hagoDeberes = true;

if(miroUnaPelicula == true && hagoDeberes == true){
    console.log("no me concentro y me pierdo buenas escenas")
}else{
    if(miroUnaPelicula != true && hagoDeberes == true){
        console.log("termino rapido deberes")
    }else{
        if(miroUnaPelicula == true && hagoDeberes != true){
            console.log("disfruto la pelicula")
        }else{
            console.log("no hago nada")
        }
    }
}

File: Ruby/01-introduccion/ejemplos-if-else.rb
# EJEMPLOS IF Y ELSE

# 1
calificacion = 5
if calificacion >= 0 && calificacion  < 4.5
  printf('reprobado')
elsif calificacion >= 4.5 && calificacion < 7
  printf('suspension')
elsif calificacion >= 7 && calificacion <=10
  printf('aprobado')
end
# 2
estalloviendo = true
nohicelatarea = true
if estalloviendo && nohicelatarea
  printf('no voy a clases')
elsif !estalloviendo && nohicelatarea
  printf('invento una escusa')
elsif estalloviendo && !nohicelatarea
  printf('llevo paraguas')
elsif !estalloviendo && !nohicelatarea
  printf('voy a clase')
end
# 3
mirounapelicula = true
hagodeberes = true
if mirounapelicula && hagodeberes
  printf('no me concentro y me pierdo buenas escenas')
elsif !mirounapelicula && hagodeberes
  printf('termino rapido deberes')
elsif mirounapelicula && !hagodeberes
  printf('disfruto la pelicula')
elsif !mirounapelicula && !hagodeberes
  printf('no hago nada')
end